src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -289,9 +289,11 @@ export default function SpeedUpAndCancelSheet() {
       startPollingGasFees(currentNetwork);
       const updateProvider = async () => {
         let provider;
-        if (currentNetwork.type === Network.mainnet && tx.flashbots) {
+        if (tx.network === Network.mainnet && tx.flashbots) {
+          logger.debug('using flashbots provider');
           provider = await getFlashbotsProvider();
         } else {
+          logger.debug('using normal provider');
```

Comment:
Do we want to keep these debug logs?

Reply:
Logs are removed in prod. Useful for debugging!